packages/front-end/components/Features/FeatureModal/EnvironmentSelect.tsx

```diff
@@ -14,32 +21,45 @@ const EnvironmentSelect: FC<{
   const { project } = useDefinitions();
   return (
     <div className="form-group">
-      <label>Enabled Environments</label>
-      <div className="appbox px-3 pt-3 pb-2 bg-light">
-        <div className="row">
+      <Container p="5" style={{ background: "#FAF8FF" }}>
+        <Text as="label" weight="bold">
+          Enabled Environments
+        </Text>
+        <CheckboxGroup.Root name="enabled-environments" mt="3">
+          <Text
+            as="label"
+            className={"rt-CheckboxItem"}
+            size="2"
+            onClick={() => {
+              environments.forEach((env) => {
+                setValue(env, true);
+              });
+            }}
+          >
+            <Flex gap="2" mb="2">
+              <RadixCheckbox checked={"indeterminate"} color={"violet"} />
+              <Flex direction="column" gap="1">
+                <Text weight="bold" className="main-text">
+                  Select All
+                </Text>
+              </Flex>
+            </Flex>
+          </Text>
```

Comment:
Used the radix component here directly for now because I wasn't sure if we wanted to support the indeterminate state on our Checkbox component. If we do, I may just need to make `setValue` optional